### DISCUSSION TITLE
Load only one language from translatable column

### DISCUSSION THREAD

**Discussion by teomanofficial** [spatie/laravel-translatable#360] [General]
I would like to ask is there any way to select only one language at a time for a localized column.

When we set a field that translatable. It loads whole translations. But for performance reasons I'ld like to load only one language at a time. 

But when I do that. When I manually extract the singular language, it throws error.

Is there any configuration for this ? I took a look at the docs but couldn't find any

**Comment by ascendeddev** [comment]
It is available by selecting  your translatable param as another param.
For example 
->select('title->en as  locale_title'])
